benchmark/Common.fs

```diff
@@ -112,7 +112,7 @@ let readNewtonsoft<'a> (stream: IO.Stream) =
         | e -> return Error (e.ToString())
     }
 
-let jsonReader<'a, 'r, 'err> (reader: Stream -> Task<Result<'a, string>>) (next: NextFunc<'a,'r, 'err>) (context: HttpContext) : HttpFuncResult<'r, 'err> =
+let jsonReader<'a, 'r, 'err> (reader: Stream -> Task<Result<'a, string>>) (next: HttpFunc<'a,'r, 'err>) (context: HttpContext) : HttpFuncResult<'r, 'err> =
```

Comment:
Should this also be changed to 'T, 'TResult, 'TError?

Reply:
Yes, haven't fixed all the testing code, benchmarks yet ...